content/en/apps/guides/security/privacy-policy.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "App Privacy Policies"
+linkTitle: "Privacy Policies"
+weight: 100
+description: >
+  Add privacy policies that users must accept before being allowed to use the app
+---
+
+As of `3.10.0`, privacy policies can be customized for every language, by adding desired content into HTML files. 
+Add these HTML files to the `privacy-policies` folder in your configuration, and associate them to the correct language in the `privacy-policies.json` file.
+```json
+{
+  "en": "en.attachment.html",
+  "fr": "fr.file.html",
+  "sw": "swahili.html",
+   ...
```

Comment:
Consider removing the `...`. I'm not sure how it shows in the docs site, but in GHs markdown it shows as invalid JSON.
```suggestion
  "sw": "swahili.html"
```